examples/lastrev-next-starter-pnpm/packages/ui/src/Card/Card.tsx

```diff
@@ -158,6 +163,12 @@ const Root = styled(MuiCard, {
   overridesResolver: (_, styles) => [styles.root]
 })<{ ownerState: CardOwnerState }>``;
 
+const CardWrap = styled(MuiCard, {
```

Comment:
Don't agree we need extra wrappers OOTB, if we are keeping these. They should be `Box`, `MuiCard` only for the root or the other way around.